zkevm-circuits/src/tx_circuit.rs

```diff
@@ -2314,6 +2480,34 @@ impl<F: Field> TxCircuitConfig<F> {
             .map(|(arg, table)| (enable.clone() * arg, table))
             .collect()
         });
+
+        ////////////////////////////////////////////////////////////////////
+        /////////////    4844: Chunk bytes RLC lookups     /////////////////
+        ///////////////// //////////////////////////////////////////////////
+        meta.lookup_any("Keccak table lookup for ChunkHash", |meta| {
+            // Isolate the last row in the fixed section, which belongs to the last tx in the chunk
```

Comment:
In the above comment (ref: `q_chunk_txbytes_hash`) I meant, you could isolate that row with a fixed column. Would make it much more easier and straightforward to understand, wdyt?

Reply:
The last row of the fixed section is now located using the fixed column `q_calldata_first`.